From 99edb13a320c30c45706a499f81601ab15b9fc2c Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Thu, 30 Sep 2021 20:41:06 -0300
Subject: [PATCH] [FIO tonxp] drm: bridge: it6161: add missing gpio consumer
 header

Fix it6161.c:2810:24: error: implicit declaration of function
'devm_gpiod_get_optional'.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 drivers/gpu/drm/bridge/it6161.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/bridge/it6161.c b/drivers/gpu/drm/bridge/it6161.c
index 1ff9ea126632e..b03e5941ef0d1 100644
--- a/drivers/gpu/drm/bridge/it6161.c
+++ b/drivers/gpu/drm/bridge/it6161.c
@@ -11,6 +11,7 @@
 #include <linux/delay.h>
 #include <linux/device.h>
 #include <linux/err.h>
+#include <linux/gpio/consumer.h>
 #include <linux/i2c.h>
 #include <linux/interrupt.h>
 #include <linux/kernel.h>
-- 
2.33.0

